packages/web-app-user-management/src/views/Users.vue

```diff
@@ -91,20 +91,23 @@
           @close="closeSideBar"
         >
           <template #body>
-            <DetailsPanel v-if="activePanel === 'DetailsPanel'" :users="selectedUsers" />
-            <EditPanel
-              v-if="activePanel === 'EditPanel'"
-              :users="selectedUsers"
-              :roles="roles"
-              @confirm="editUser"
-            />
-            <GroupAssignmentsPanel
-              v-if="activePanel === 'GroupAssignmentsPanel'"
-              :users="selectedUsers"
-              :roles="roles"
-              :groups="groups"
-              @confirm="editUserGroupAssignments"
-            />
+            <oc-loader v-if="loadAdditionalUserDataTask.isRunning" />
```

Comment:
```suggestion
            <oc-loader v-if="loadAdditionalUserDataTask.isRunning || !loadAdditionalUserDataTask.last" />
```
If the task is not started `isRunning` evaluates to `false`.

For better readability you could also introduce a new computed prop.